### DISCUSSION TITLE
How to disable rules (or completely turn off rome linter) for a file

### DISCUSSION THREAD

**Discussion by llc1123** [rome/tools#3792] [Q&A] [answered]
I often use something like `/* eslint-disable @typescript-eslint/no-explicit-any */` in some files, and I cannot find a replacement in rome linter.

**Comment by MichaReiser** [comment, marked as answer]
Hey @llc1123 

Rome doesn't support this today, but we'll hopefully be supported soon. Do you want to create an issue for adding support for file level suppressions so that it's on our radar?
